### ISSUE TITLE
Adb failed to reboot!, reboot failed: -1 : 

### ISSUE THREAD

**Issue by gabriel-lucas** [ubports/ubports-installer#195]
Automatically generated error report 
Version: 0.1.12-beta 
Host OS: linux 
Host Dist: elementary OS undefined
Host Arch: x64 
Node: v8.2.1 

Error log: http://paste.ubuntu.com/p/CRD7HQrrJv/

**Comment by NeoTheThird** [comment]
This should have been fixed by #160. Any chance you could try that again w/ the source from master?

**Comment by gsilvapt** [comment]
@NeoTheThird I believe that change hasn't yet been released so to test that change one has to use the source code. Am I wrong?

**Comment by NeoTheThird** [comment]
@gsilvapt You are right: https://github.com/ubports/ubports-installer/compare/0.1.12-beta...master

Consult README.md for instructions to run this from source.
